src/plugin/ipc/socket/socketconnection.cpp

```diff
@@ -178,9 +210,6 @@ void SocketConnection::serialize(jalib::JBinarySerializer& o)
   TcpConnection::TcpConnection(int domain, int type, int protocol)
   : Connection(TCP_CREATED)
   , SocketConnection(domain, type, protocol)
```

Comment:
The fields have been moved to the super class, `SocketConnection`, because I felt that these are common to all socket types. They will be initialized to the default values in the constructor of `SocketConnection`.